models/staging/commissioning/sus/stg_ecds_consolidated.sql

```
@@ -0,0 +1,327 @@
+						) ct
+	ON att.PRIMARYKEY_ID = ct.PRIMARYKEY_ID
+
+LEFT JOIN
+	{{ref('raw_sus_ae_patient_mental_health_act_legal_status')}} AS mh
+	ON att.PRIMARYKEY_ID = mh.PRIMARYKEY_ID
+		AND mh.MENTAL_HEALTH_ACT_LEGAL_STATUS_ID = 1
+		--AND mh.MENTAL_HEALTH_ACT_LEGAL_STATUS_ID = (SELECT MIN(mh.MENTAL_HEALTH_ACT_LEGAL_STATUS_ID) FROM [dmic_ECDS].[ECDS].[patient.mental_health_act_legal_status] AS mh WITH (NOLOCK) 
+			--WHERE att.PRIMARYKEY_ID = mh.PRIMARYKEY_ID)
+LEFT JOIN (
+				SELECT
+				*
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)